netqasm/sdk/builder.py

```diff
@@ -1680,20 +1570,25 @@ def sdk_epr_rsp_recv(
             qubit_ids_array, ent_results_array, wait_all, params
         )
 
-        return qubit_futures
+        epr_results = deserialize_epr_keep_results(params.number, ent_results_array)
+        return qubit_futures, epr_results
 
-    def sdk_create_epr_keep(self, params: EntRequestParams) -> List[Qubit]:
+    def sdk_create_epr_keep(
```

Comment:
Public methods should have docstring. We generate read the doc for this don't we? Some other methods also in this module have no docstring

Reply:
added docstrings (also for other methods)